lib/redmine_pulls/patches/adapters/git_adapter_helper.rb

```diff
@@ -69,6 +94,14 @@ def is_ancestor?(expected_ancestor, expected_descendant)
 
             ancestor_revision == merge_base
           end
+
+          def git_cmd_with_output(command)
```

Comment:
- I don't like the `_with_` here. I think it would be clearer if the name is only `git_cmd_output`.
- This method should be declared as a `private` method
- Can you add the argument `options = {}` and pass it to the `git_cmd`. We don't use it now, but it could be useful in the future.